### ISSUE TITLE
Unable to use the Vacuum tile

### ISSUE THREAD

**Issue by DeviousPenguin** [resoai/TileBoard#219]
Hi, I'm trying to use the Vacuum tile but it's just not working for me.

There's no example tile on the examples page, so I tried to put the entity id in the tile like this

```
{
  position: [3, 2],
  type: TYPES.VACUUM,
  id: 'vacuum.auto',
},
```
and it shows the current status, however none of the controls appear. How can I fix this?
Thanks

**Comment by resoai** [comment]
You can only start and stop the vacuum via the tile.

**Comment by DeviousPenguin** [comment]
Ah so the entire tile starts and stops the vacuum in the same way as switch tile?

I assumed it was going to be like the media player tile with play/pause/stop/return home.

I'm happy with start/stop but the only problem when I click the tile I only get an error message:

`Error handling message: required key not provided @ data['service']. Got None`

Here's a screenshot: 
![vacuum_error_crop](https://user-images.githubusercontent.com/11376139/55722320-d06e5380-59fd-11e9-8754-73bfc3a07348.png)

Do I need to add something to the tile to tell it which service it should call on click?

**Comment by resoai** [comment]
Nope, the vacuum tile is just a glorified switch which was blindly developed from the data gathered from other people, since I don't have one. What are the names of the services for "start" and "stop" for your vacuum?

**Comment by DeviousPenguin** [comment]
I use servce: **vacuum.start** and **service: vacuum.return_to_base** to start and stop the vacuum, however I should have mentioned sooner that my 'vacuum' is actually an automower, it's just that HA doesn't have an automower device so the addon uses the vacuum platform in HA.

I've actually managed to get it working in HA by using the template switch:

https://www.home-assistant.io/components/switch.template/

And then just adding it in as a normal switch in Tileboard. That way I'm able to get the state of the switch using jinja based on the state of the mower, as it has multiple start and stopped states using the value_template.

If I could make a suggestion for the Vacuum tile it would be that the on and off services could somehow be set in the config.js of Tileboard, in a similar way icons can be set of switch on and off states. It's a shame there isn't an easier way to test these devices without actually having them, I can't really think of a way around that.

I'll close this now as this issue has been solved using the switch template. :)
